x-pack/plugins/ingest_manager/server/services/epm/kibana/index_pattern/install.ts

```diff
@@ -386,9 +388,9 @@ export const ensureDefaultIndices = async (callCluster: CallESAsCurrentUser) =>
             },
           });
         } catch (putErr) {
-          // throw new Error(`${defaultIndexPatternName} could not be created`);
-          throw new Error(putErr);
+          logger.error(`${defaultIndexPatternName} could not be created`);
```

Comment:
When the logger is indeed undefined, won't this result in an error?

Reply:
We get in the plugin constructor and pass it to appContextService in the plugin start().  In what case would it be undefined?